pkgs/applications/virtualization/OVMF/default.nix

```diff
@@ -1,8 +1,23 @@
 { stdenv, nixosTests, lib, edk2, util-linux, nasm, acpica-tools, llvmPackages
+, fetchurl, python3, pexpect, xorriso, qemu, dosfstools, mtools
 , csmSupport ? false, seabios
 , fdSize2MB ? csmSupport
-, fdSize4MB ? false
+, fdSize4MB ? secureBoot
```

Comment:
Does this change the default fd size for OVMFFull? I am having a hard time telling from the edk2 repo what the default is.

Reply:
Yes, it does. I believe the default is quite small and, according to Debian, insufficient for secure boot

Reply:
Yeah looks like it's 1MB by default. Are there any negative implications you know of with changing this default? https://github.com/tianocore/edk2/blob/master/OvmfPkg/README

Reply:
My understanding, which is sketchy, is that this just increases the (maximum?) size of the image on disk and of course gives more room for NVRAM data